bitnami/external-dns/templates/_helpers.tpl

```diff
@@ -190,7 +190,10 @@ Return the name of the Secret used to store the passwords
   "accessKeyId": "{{ .Values.alibabacloud.accessKeyId }}",
   {{- end}}
   {{- if .Values.alibabacloud.accessKeySecret }}
-  "accessKeySecret": "{{ .Values.alibabacloud.accessKeySecret }}"
+  "accessKeySecret": "{{ .Values.alibabacloud.accessKeySecret }}",
+  {{- end}}
+  {{- if .Values.alibabacloud.vpcId }}
+  "vpcId": "{{ .Values.alibabacloud.vpcId }}"
   {{- end}}
```

Comment:
Could you add this property before `accessSecretKey`? I.e. right after `regionId`. I'm a bit concerned with the comma (",") as it may be that `vpcId` is not set while `accessSecretKey` is, causing a possible failure (?)

Reply:
make sense